CONTRIBUTING.md

```diff
@@ -102,13 +166,30 @@ GitHub web notifications using a tool such as [Octobox](http://octobox.io/).
 
 ## Contributing outside this repository
 
-There are many useful ways to contribute to the Coq ecosystem that don't involve the Coq repository.
-
-Tutorials to teach Coq, and especially to teach particular advanced features, would be much appreciated. Some tutorials are listed on the [Coq website](https://coq.inria.fr/documentation). If you would like to add a link to this list, please make a pull request against the Coq website repository at https://github.com/coq/www.
-
-External plugins / libraries contribute to create a successful ecosystem around Coq. If your external development is mature enough, you may consider submitting it for addition to our CI tests. Refer to [`dev/ci/README.md`](/dev/ci/README.md) for more information.
-
-Ask and answer questions on [Stack Exchange](https://stackexchange.com/filters/299857/questions-tagged-coq-on-stackexchange-sites) which has a helpful community of Coq users.
+There are many useful ways to contribute to the Coq ecosystem that don't
+involve the Coq repository.
+
+Tutorials to teach Coq, and especially to teach particular advanced features,
+are much appreciated. Some tutorials are listed on the
+[Coq website](https://coq.inria.fr/documentation). If you would like to add
+a link to this list, please make a pull request against the Coq website
+repository at <https://github.com/coq/www>.
+
+External plugins / libraries contribute to create a successful ecosystem
+around Coq. If your external development is mature enough, you may consider
+submitting it for addition to our CI tests. Refer to
+[`dev/ci/README-users.md`](dev/ci/README-users.md) for more information.
+
+Some Coq packages are not maintained by their authors anymore even if they
+were useful (for instance because they changed jobs). The coq-community
+organization is a place for people to take over the maintenance of such
+useful packages. If you want to contribute by becoming a maintainer, you can
+find a list of packages waiting for a maintainer [here](https://github.com/coq-community/manifesto/issues?q=is%3Aissue+is%3Aopen+label%3Amaintainer-wanted).
+You can also propose a package that is not listed. Find out more about
+coq-community in [the manifesto's README](https://github.com/coq-community/manifesto).
+
+Ask and answer questions on [Stack Exchange](https://stackexchange.com/filters/299857/questions-tagged-coq-on-stackexchange-sites)
```

Comment:
IMO it is strange that you don't advertise discourse now that you are refreshing the guide; IMO no harm on having it now, and then we won't forget to have it later.

Reply:
I agree that this is a good time to put it there.